html_layout.py

```diff
@@ -325,7 +442,19 @@ def create_appLayout(
                                     )
                                 ],
                                 className="box-fields"
-                            )
+                            ),
+
+                            html.Div(
+                                [
+                                    html.Div('i', className='tooltip-icon'),
+                                    html.P(
+                                        "The following two fields are used to define the PUE and energy mix associated with your computations.",
```

Comment:
Is there a way to make this dropdown and the ones below the same width? The (i) seems to be squishing this one (maybe add a "ghost" (i) div to the ones below)